Sources/Benchmarks/main.swift

```diff
@@ -0,0 +1,3 @@
+try benchmarkBSONEncoding()
```

Comment:
One thing that is missing is the computation of the ["composite scores"](https://github.com/mongodb/specifications/blob/master/source/benchmarking/benchmarking.rst#composite-score-calculation). Will that be something that gets added later after the parallel tests?